lib/qa/authorities/linked_data/config.rb

```diff
@@ -44,11 +44,20 @@ def prefixes
         @prefixes ||= authority_config.fetch(:prefixes, {})
       end
 
+      def config_version
+        @config_version ||= authority_config.fetch(:QaConfigVersion, '1.0')
+      end
+
+      def config_version?(version)
+        config_version == version
+      end
+
       # Return the full configuration for an authority
       # @return [String] the authority configuration
       def authority_config
         @authority_config ||= Qa::LinkedData::AuthorityService.authority_config(@authority_name)
         raise Qa::InvalidLinkedDataAuthority, "Unable to initialize linked data authority '#{@authority_name}'" if @authority_config.nil?
+        convert_1_0_to_2_0 if @authority_config.fetch(:QA_CONFIG_VERSION, '1.0') == '1.0'
```

Comment:
Automatically convert 1.0 versioned configs (default version if unspecified) to 2.0 version compliance.  Provides backward compatibility of existing configs.